gulpfile.js

```diff
@@ -247,6 +250,23 @@ gulp.task("link-bugs", function (done) {
     exec(bin("linkify-bugs.sh"), [path.join(SOURCE_DIR, "_posts")], done);
 });
 
+gulp.task('lint', function() {
```

Comment:
Ideally, build should use this task and not succeed if linting fails. How about `jslint`?